PaulasWeatherApp/PaulasWeatherApp/ViewController/CurrentWeatherViewController.swift

```diff
@@ -8,11 +8,104 @@
+        activityIndicator.center = self.view.center
+        activityIndicator.hidesWhenStopped = true
+        self.view.addSubview(activityIndicator)
+        
+        activityIndicator.startAnimating()
+        setCurrentWeatherViews()
         view.backgroundColor = .systemRed
     }
     
+    func setCurrentWeatherViews(){
+        
+        tempLabel.translatesAutoresizingMaskIntoConstraints = false
+        tempLabel.textAlignment = .center
+        tempLabel.font = .systemFont(ofSize: 36)
+        tempLabel.textColor = .black
+        tempLabel.text = "... °C"
+        tempLabel.backgroundColor = .clear
+        tempLabel.sizeToFit()
+        
+        sunriseLabel.translatesAutoresizingMaskIntoConstraints = false
+        sunriseLabel.textAlignment = .center
+        sunriseLabel.font = .systemFont(ofSize: 12)
+        sunriseLabel.textColor = .black
+        sunriseLabel.backgroundColor = .clear
+        sunriseLabel.attributedText = sunriseString
+        sunriseLabel.sizeToFit()
+        
+        sunsetLabel.translatesAutoresizingMaskIntoConstraints = false
+        sunsetLabel.textAlignment = .center
+        sunsetLabel.font = .systemFont(ofSize: 12)
+        sunsetLabel.textColor = .black
+        sunsetLabel.backgroundColor = .clear
+        sunsetLabel.attributedText = sunsetString
+        sunsetLabel.sizeToFit()
+        
+        self.view.addSubview(tempLabel)
+        self.view.addSubview(sunsetLabel)
+        self.view.addSubview(sunriseLabel)
+        
+        NSLayoutConstraint.activate([
+            tempLabel.centerXAnchor.constraint(equalTo: view.centerXAnchor),
+            tempLabel.centerYAnchor.constraint(equalTo: view.centerYAnchor, constant: -70),
+            
+            sunriseLabel.leadingAnchor.constraint(equalTo: tempLabel.leadingAnchor),
+            sunriseLabel.topAnchor.constraint(equalTo: tempLabel.bottomAnchor),
+            
+            sunsetLabel.leadingAnchor.constraint(equalTo: sunriseLabel.trailingAnchor,constant: 10),
+            sunsetLabel.trailingAnchor.constraint(equalTo: tempLabel.trailingAnchor),
+            sunsetLabel.topAnchor.constraint(equalTo: tempLabel.bottomAnchor),
+            
+            ])
+    }
+    
+    func setCurrentWeatherData(_ data : WeatherDataModel) {
+        let dateFormatter = DateFormatter()
+        dateFormatter.dateFormat = "hh:mm a"
+        
+        tempLabel.text = "\(data.current.temp)  °C"
+        guard let sunset = data.current.sunset else { return }
+        guard let sunrise = data.current.sunrise else { return }
+        
+        sunsetString.append(NSAttributedString(string: dateFormatter.string(from: Date(timeIntervalSince1970: Double(sunset))) ))
+        sunsetLabel.attributedText = sunsetString
+        
+        sunriseString.append(NSAttributedString(string: dateFormatter.string(from: Date(timeIntervalSince1970:Double(sunrise)))))
+        sunriseLabel.attributedText = sunriseString
+
+        
+        
+       
```

Comment:
Pay attention to coding style. These are extra lines that are not needed. a good place to read more about this topic is https://confluence.garmin.com/display/DI/iOS+-+Coding+standards+Swift